samples/CallWithChat/src/app/views/CallScreen.tsx

```diff
@@ -78,7 +79,12 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
         tooltipText: 'Custom Background'
       }
     ];
-    return { videoBackgroundImages: videoBackgroundImages };
+    return {
+      videoBackgroundOptions: {
+        videoBackgroundImages,
+        onResolveDependency: onResolveVideoEffectDependencyLazy
```

Comment:
We'll need UI tests with and without this

Reply:
I don't see a UI test update that no longer shows the effects button in the call screen camera button. Are we missing that test or does the effects button incorrectly show?